values/src/main/java/org/spine3/net/Urls.java

```diff
@@ -41,19 +53,43 @@ public static Url of(Url rawUrl) {
         return url;
     }
 
+    /**
+     * Converts String URL representation into {@link Url} instance.
```

Comment:
Please mention that no validation of the value occurs. People may think we do the same that standard Java `URL` class does.